### DISCUSSION TITLE
Issue, soapysdrutil is missing

### DISCUSSION THREAD

**Discussion by pierre-olivier832** [cariboulabs/cariboulite#102] [Q&A]
Hi, during the installation, it stop because theres missing soapysdrutil.
I want to run it on an industrial pi pannelview ![20230329_002723.jpg](https://user-images.githubusercontent.com/79736604/228426817-a3bca06a-b831-48b3-b1a7-b5e01a59e390.jpg)

**Comment by McGr** [comment]
I had the same issue.
`sudo apt-get install soapyremote-server`
fixed it for me

**Reply by pierre-olivier832** [reply]
I always have the same issue

**Reply by pierre-olivier832** [reply]
I try install.sh and i have the same problem.  Does we need to write bash install.sh?

**Comment by konimaru** [comment]
The primary repository is - for want of a better word - _difficult to use_. Head over to #82 and see how far you get.

**Reply by pierre-olivier832** [reply]
![20230410_192603.jpg](https://user-images.githubusercontent.com/79736604/231018136-330b9222-a789-4b39-a32c-a106ec3d3a37.jpg)

I try to boot dragonos on a raspberry pi zero.
